README.md

```diff
@@ -1,4 +1,4 @@
-# RLCard: A Toolkit for Reinforcement Learning in Card Games
+# RLCard: A Toolkit for Reinforcement Learning in Card GameS
```

Comment:
Why use capital <kbd>s</kbd>?

Reply:
Sorry, where is this README.md located? I did not encounter it.